src/sentry/snuba/metrics/query.py

```diff
@@ -88,6 +88,16 @@ class MetricsQuery(MetricsQueryValidationRunner):
     histogram_from: Optional[float] = None
     histogram_to: Optional[float] = None
 
+    @staticmethod
+    def _use_case_id(metric_mri: str) -> UseCaseKey:
+        parsed_mri = parse_mri(metric_mri)
+        # Find correct use_case_id based on metric_name
+        if parsed_mri is not None and parsed_mri.namespace == "transactions":
```

Comment:
Can't we unify this `if parsed_mri is not None` as a root level condition? think it would make it better rather than repeating the condition

Reply:
will do.

Reply:
@ahmedetefy fixed.